Include/internal/pycore_function.h

```diff
@@ -16,7 +16,7 @@ extern PyObject* _PyFunction_Vectorcall(
 
 #define FUNC_MAX_WATCHERS 8
 
-#define FUNC_VERSION_CACHE_SIZE (1<<12)  /* Must be a power of 2 */
+#define FUNC_VERSION_CACHE_SIZE (1<<14)  /* Must be a power of 2 */
```

Comment:
This change seems unrelated, and probably isn't the best way to fix the underlying issue.

Reply:
The change was because the function cache was too small. This was causing the tracer/optimizer to bail too early because it couldn't find the function when it tried to lookup the function version. In some cases, the function is actually still there, it's just no in the version cache. It's an actual problem.

Reply:
It might be affecting the stats though, because this might lead more things to be executed in tier 2. I will revert it now and revisit it later.